src/index.ts

```diff
@@ -23,6 +23,11 @@ export const beautifier: Beautifier = {
       name: "ESLint",
       package: "eslint",
     },
+    {
+      type: DependencyType.Node,
+      name: "ESLintPluginReact",
```

Comment:
`ESLint-plugin-React` (as seen in https://www.npmjs.com/package/eslint-plugin-react ) may be a better `name`.

Reply:
Good approach, but what about on L60, where we assign it to a variable. There I need to call it `ESLintPluginReact` and then we would have two different styles.

Reply:
Yup, change line 60, too, so they are the same 👍 .